apps/site/pages/components/molecules/region-bar.mdx

```diff
@@ -0,0 +1,161 @@
+---
+description: The Region Bar is a similar proposal for the Region Button, usually used on mobile devices only. It triggers the RegionModal.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Button___6d6b16886d5d7e2a05dba8c45c075796.png
+---
+
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { RegionBar } from '@faststore/ui'
+
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const regionBarPath = path.resolve(__filename)
+  const components = ['RegionBar.tsx']
+  const [regionBarProps] = getComponentPropsFrom(regionBarPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: { regionBarProps },
+    },
+  }
+}
+
+export const RegionBarPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { regionBarProps } = useSSG()
+  return {
+    RegionBar: <PropsSection propsList={regionBarProps} />,
+  }[component]
+}
+
+<header>
+
+# Region Bar
+
+The Region Bar is a similar proposal for the Region Button, usually used on mobile devices only. It should trigger the RegionModal.
+
+The component is part of [Region](/components/region) feature.
+
+</header>
+
+## Overview
+
+<OverviewSection>
+  <RegionBar onButtonClick={() => {}} />
+  <RegionBar postalCode={'151515'} />
+</OverviewSection>
+```tsx
+<RegionBar onButtonClick={() => {}} />
+<RegionBar postalCode={'151515'} />
+```
+
+---
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
```

Comment:
This documentation don't use tabs on purpose?

Reply:
yeah. Because it's just one line.